eng/.docsettings.yml

```diff
@@ -67,77 +68,65 @@ known_content_issues:
       "sdk/cognitiveservices/cognitiveservices-anomalydetector/README.md",
        "#1583",
     ]
-  - ["sdk/cognitiveservices/cognitiveservices-autosuggest/README.md",  "#1583"]
+  - ["sdk/cognitiveservices/cognitiveservices-autosuggest/README.md",  "#895"]
```

Comment:
This also looks like a closed PR: https://github.com/Azure/azure-sdk-for-js/pull/895

Reply:
I should have checked the appropriate repo in here - https://github.com/Azure/autorest.typescript/pull/895 
I'll update it. Thanks!

Reply:
done!